packages/elements/src/components/ui/buttons/input-field/InputFieldButton.module.css

```diff
@@ -0,0 +1,49 @@
+.inputFieldButton {
+  --swui-inputfieldbutton-width: 24px;
+  --swui-inputfieldbutton-height: 24px;
+  --swui-inputfieldbutton-margin: 4px;
```

Comment:
Don't use margin. A component should never layout/align itself, that is always the responsibility of the parent.